config/main.py

```diff
@@ -5179,6 +5179,16 @@ def enable(ctx):
         ctx.fail("Unable to check sflow status {}".format(e))
 
     if out != "active":
+        # check if service is masked, if yes, unmask it
```

Comment:
better to check if the feature is enabled or not, if not then print out message ask user to enable the feature first.

Reply:
Sure, hostcfgd will unmask it anyways during feature enable. So I will check for feature enable and echo the msg to user. Thx.